cypress.config.js

```diff
@@ -0,0 +1,20 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+    e2e: {
+        /**
+         * Override baseUrl if provided via the config.env.json
```

Comment:
```suggestion
         * Override baseUrl if provided via the cypress.env.json
```
?

Reply:
done

Reply:
Sorry was referring to the filename in the comment, not the variable name which is correctly `config`.

Reply:
Reverted and updated the comment